cudaaligner/src/matrix_cpu.hpp

```diff
@@ -34,7 +34,7 @@ class matrix
 {
 public:
     using value_type = T;
-    static inline bool error(int t)
+    static inline bool error(int32_t t)
```

Comment:
If you use fixed width integers, we usually also include `<cstdint>` then

Reply:
Good point.

Reply:
fixed.